@here/harp-omv-datasource/lib/OmvDecodedTileEmitter.ts

```diff
@@ -137,6 +137,38 @@ const MAX_CORNER_ANGLE = Math.PI / 8;
  */
 const INVALID_ARRAY_INDEX = -1;
 
+function maxValue(array: ArrayLike<number>): number {
+    let max = -Infinity;
+
+    // tslint:disable-next-line: prefer-for-of
+    for (let i = 0; i < array.length; ++i) {
+        if (array[i] > max) {
+            max = array[i];
+        }
+    }
+
+    return max;
+}
+
+function createIndexBuffer(elements: ArrayLike<number>) {
+    return maxValue(elements) > 65535 ? new Uint32Array(elements) : new Uint16Array(elements);
+}
+
+function createIndexBufferAttribute(
+    elements: ArrayLike<number>,
+    name: string = "index"
+): BufferAttribute {
+    const type = maxValue(elements) > 65535 ? "uint32" : "uint16";
```

Comment:
In native we even used 16 bit when exceeding 65K and did multiple draw calls with [glDrawElementsBaseVertex.](https://www.khronos.org/registry/OpenGL-Refpages/gl4/html/glDrawElementsBaseVertex.xhtml).
Not sure if sth. equivalent exists in WebGL.

Reply:
I think there is only this extension for WebGL 2: https://www.khronos.org/registry/webgl/extensions/WEBGL_draw_instanced_base_vertex_base_instance/. So not very likely that this is supported by many devices.